app/views/handlers/question.py

```diff
@@ -154,21 +158,22 @@ def _is_list_just_primary(self, list_items, list_name):
         )
 
     def _get_answer_action(self):
-        answers = self.rendered_block["question"]["answers"]
-
-        for answer in answers:
-            # pylint: disable=no-member
-            # wtforms Form parents are not discoverable in the 2.3.3 implementation
-            submitted_answer = self.form.data[answer["id"]]
-
-            for option in answer.get("options", {}):
-                action = option.get("action")
-
-                if action and (
-                    option["value"] == submitted_answer
-                    or option["value"] in submitted_answer
-                ):
-                    return action
+        if self.rendered_block.get("question"):
+            answers = self.rendered_block["question"]["answers"]
+
+            for answer in answers:
+                # pylint: disable=no-member
+                # wtforms Form parents are not discoverable in the 2.3.3 implementation
+                submitted_answer = self.form.data[answer["id"]]
+
+                for option in answer.get("options", {}):
+                    action = option.get("action")
+
+                    if action and (
+                        option["value"] == submitted_answer
+                        or option["value"] in submitted_answer
+                    ):
+                        return action
```

Comment:
We should avoid this deep nesting but investing the logic and existing early. Do you even need this change once the change to handle post are done to call the parent of Question?

Reply:
This block question check is still needed, so just returning early now and nesting is reduced.